modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerExtension.java

```diff
@@ -71,6 +81,7 @@ public void setVersion(String version) {
 	private Object _baseDir = "poshi-tests";
 	private String _openCVVersion = "2.4.9-0.9";
 	private final Map<String, Object> _poshiProperties = new HashMap<>();
+	private Object _poshiPropertiesFile;
```

Comment:
Maybe we can readd the default value `poshi-runner.properties`, so people will be able to create that file, and everything will work without any configuration.